compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion/MetadataEncoding.hs

```diff
@@ -459,7 +459,7 @@ natSynTCon = LF.Qualified
     , qualObject = LF.TypeConName [tconName]
     }
     where
-        packageId = "38e6274601b21d7202bb995bc5ec147decda5a01b68d57dda422425038772af7"
+        packageId = "eb6926e50bb83fbc8f3e154c7c88b1219b31a3c0b812f26b276d46e212c2dd71"
```

Comment:
Yeah, me too! I think that's because the engine accepts both.

Reply:
oh, I think we missed it because the data-dependencies test case `Type synonyms over data-dependencies` is defined using `simpleImportTest`, which only tests the default target LF :( oh well, at least it will get tested now